### DISCUSSION TITLE
Colorizing warnings/errors breaks ::group

### DISCUSSION THREAD

**Discussion by wipe2238** [community/community#25549] [Actions]
<pre><code>group-bug:
  runs-on: windows-latest
  steps:

  - run: |
           :
           echo ---
           echo Warning single
           echo Error single
           echo [Warning] single
           echo [Error] single
           echo ---
           echo ::group::Warning grouped
           echo grouped
           echo warnings
           echo ::endgroup::
           echo ---
           echo ::group::Error grouped
           echo grouped
           echo errors
           echo ::endgroup::
           echo ---
           echo ::group::[Warning] grouped
           echo grouped
           echo warnings
           echo ::endgroup::
           echo ---
           echo ::group::[Error] grouped
           echo grouped
           echo errors
           echo ::endgroup::
           echo ---
    shell: bash
</code></pre>
<p><img src="https://i.imgur.com/auivm8Y.png" alt=""></p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Command “echo ::group::[Warning] grouped” should be not supported.</p>
<pre><code>echo "::group::my title"  
echo "::endgroup::"
</code></pre>
<p>This should be wrapped by the core methods which comes from ‘<a href="https://github.com/actions/toolkit/blob/master/docs/commands.md#group-and-ungroup-log-lines" rel="nofollow noopener">actions/toolkit</a>’ , it requires a string value and but not support nested warpped “[Warning] title”.</p>
<p>code as below:</p>
<pre><code>function startGroup(name: string): void {}
function endGroup(): void {}
</code></pre>
<p>Please refer to official doc for more details: </p>
<p><a href="https://github.com/actions/toolkit/blob/master/docs/commands.md#group-and-ungroup-log-lines" rel="nofollow noopener">https://github.com/actions/toolkit/blob/master/docs/commands.md#group-and-ungroup-log-lines</a></p>
